extensions/common/json-ld/src/main/java/org/eclipse/edc/jsonld/JsonLdExtension.java

```diff
@@ -89,9 +90,13 @@ public JsonLd createJsonLdService(ServiceExtensionContext context) {
         }
         service.registerNamespace(EDC_PREFIX, EDC_NAMESPACE);
 
-        getResourceUri("document" + File.separator + "odrl.jsonld")
-                .onSuccess(uri -> service.registerCachedDocument("http://www.w3.org/ns/odrl.jsonld", uri))
-                .onFailure(failure -> monitor.warning("Failed to register cached json-ld document: " + failure.getFailureDetail()));
+        Stream.of(
+                new JsonLdContext("odrl.jsonld", "http://www.w3.org/ns/odrl.jsonld"),
+                new JsonLdContext("dspace.jsonld", "https://w3id.org/dspace/2024/1/context.json")
```

Comment:
I think we will need also a local copy of the `dspace.jsonld` in the `resources/document` folder

Reply:
woa, totally slipped off!